compiler/front/depfiles.nim

```diff
@@ -0,0 +1,41 @@
+#
+#
+#           The Nim Compiler
+#        (c) Copyright 2015 Andreas Rumpf
```

Comment:
I missed this, sorry. Please remove these two lines.

After that it's good to go.